core/driver/tests/src/test/resources/CustomCommand/ideprobe.conf

```diff
@@ -2,7 +2,7 @@ probe {
   workspace.path = "classpath:/CustomCommand"
 
   driver {
-    headless = false
```

Comment:
it was false, so it should be "native" or empty, right?

Reply:
yes, my mistake - just fixed it